scandium-core/src/main/java/org/eclipse/californium/scandium/dtls/DTLSSession.java

```diff
@@ -664,7 +664,7 @@ public Principal getPeerIdentity() {
 	/**
 	 * Sets the authenticated peer's identity.
 	 * 
-	 * @param the identity
+	 * @param peerIdentity the identity
```

Comment:
this doesn't seem to be related to the other changes.
next time, please create a separate PR for it